README.md

```diff
@@ -0,0 +1,60 @@
+# Code behind **Data Analytics at Nesta** Medium Articles :wave:
+
+<img width="520" alt="Screenshot 2022-10-19 at 12 53 30" src="https://user-images.githubusercontent.com/46863334/196672092-d7b07034-4078-473d-9fb3-4992c97b71b4.png">
+
+**A place to store code for [Medium articles written by Nesta's Data Analytics Team.](https://medium.com/data-analytics-at-nesta)**
+
+This repository is to store code for Medium articles written by Nesta's Data Analytics Team. The purpose of our Medium is to share ideas, reflections and tips and tricks for doing data science for social good. Where the code is soley to compliment a given medium article, this is where you should put it. 
+
+## ❓ Is this the right place for my project? 
+
+1. If the original purpose of the code was to serve a medium blog, it should live in the medium repo. The readme of that repo should be a sort of table of contents of code used in blogs.
+2. If the blog is referencing code from a project repo, then add a link to that in the medium repo readme
+3. If the code you've made also provides material for an internal tutorial then it's arguable that it should live in its own place, as there's a chance it will get updated.
+
+Long story short - if you're not going to update the code and it's simply to compliment an article, this is the repo for you! The idea is that the projects should be self-contained, so think Google Co-lab style code sharing.  
+
+## 📖 Directory
+
+**< Project name (with link to sub-directory) >**
+< Short description of project and links to any key resources >
+
+**[spancat tutorial](https://github.com/nestauk/dap_medium_articles/tree/dev/spancat_tutorial)**
+A tutorial to train spaCy's spancat component without using config based training. Read the [Medium article here](https://medium.com/p/992024d047c2/edit) and follow along with the [jupyter notebook here](https://github.com/nestauk/dap_medium_articles/blob/dev/spancat_tutorial/spancat_training.ipynb). 
+
+## 🎮 User guidelines
+
+1. Use the directory above to find the code behind an article that you are looking for.
+2. Clone this repository.
+3. Follow the code behind the article README for any environment set up (if relevant). 
```

Comment:
Maybe this should say something more like "navigate to the sub directory of the post you're interested in and checkout the readme for running instructions"